src/Pelagos/Util/UrlValidation.php

```diff
@@ -20,8 +20,12 @@ class UrlValidation
     public function validateUrl($url)
     {
         $ch = curl_init();
+        $cookies = tempnam('/tmp','cookie.txt');
+        curl_setopt($ch, CURLOPT_COOKIEJAR, $cookies);
```

Comment:
Instead of giving it a cookie file, you can use '-' which means output to standard out. (Even if you don't use it.)

Reply:
Or even better use: 'php://memory' as file name.

Reply:
Used php:://memory. 👍